### DISCUSSION TITLE
Search option on Chakra UI website for easy navigation

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#2568] [Ideas] [answered]
It would help to add a search bar in the top menu or sidebar of Chakra UI's website.  Sifting through the docs gets tricky and a little tedious when trying to execute search queries.  Currently, I have to resort to google, which links the appropriate page.

**Comment by chasinhues** [comment, marked as answer]
I believe this is on the Chakra team's todo list, but they're waiting for their search provider algolia to re-index the new website

**Reply by AlonGvili** [reply]
any update on this ?
my goal is to add a search query into ms edge so i can just do ```chui + tab``` and start a search in the chakra-ui docs
